challenges/objects-arrays.js

```diff
@@ -49,22 +77,30 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+
+const universities = graduates.map((graduates) => {
+  return graduates.university;
+});
+console.log (universities.sort());
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
-const contactInfo = [];
+const contactInfo = graduates.map((graduates) => {
+  return `${graduates.first_name} ${graduates.email}`;
+});
```

Comment:
don't forget to rename the element to something other than the name of the array.